<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fourier Transform</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,10,10,10">
    <item row="4" column="0">
     <widget class="QCustomPlot" name="resultGraph" native="true"/>
    </item>
    <item row="3" column="0">
     <widget class="QCustomPlot" name="walshGraph" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QCustomPlot" name="sourceGraph" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,50,0,0,0,7,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>y =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="equationText">
         <property name="text">
          <string>sin(2*x)+cos(7*x)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Hadamard -&gt; Walsh</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>N = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sampleSizeText">
         <property name="text">
          <string>64</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Discrete</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fast</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
